integration-cli/docker_cli_build_test.go

```diff
@@ -4048,43 +4048,8 @@ func (s *DockerSuite) TestBuildRUNErrMsg(c *check.C) {
 	})
 }
 
-func (s *DockerTrustSuite) TestTrustedBuild(c *check.C) {
-	repoName := s.setupTrustedImage(c, "trusted-build")
-	dockerFile := fmt.Sprintf(`
-  FROM %s
-  RUN []
-    `, repoName)
-
-	name := "testtrustedbuild"
-
-	buildImage(name, trustedBuild, build.WithDockerfile(dockerFile)).Assert(c, icmd.Expected{
-		Out: fmt.Sprintf("FROM %s@sha", repoName[:len(repoName)-7]),
-	})
-
-	// We should also have a tag reference for the image.
-	dockerCmd(c, "inspect", repoName)
-
-	// We should now be able to remove the tag reference.
-	dockerCmd(c, "rmi", repoName)
-}
-
-func (s *DockerTrustSuite) TestTrustedBuildUntrustedTag(c *check.C) {
-	repoName := fmt.Sprintf("%v/dockercli/build-untrusted-tag:latest", privateRegistryURL)
-	dockerFile := fmt.Sprintf(`
-  FROM %s
-  RUN []
-    `, repoName)
-
-	name := "testtrustedbuilduntrustedtag"
-
-	buildImage(name, trustedBuild, build.WithDockerfile(dockerFile)).Assert(c, icmd.Expected{
-		ExitCode: 1,
-		Err:      "does not have trust data for",
-	})
-}
-
 // FIXME(vdemeester) should migrate to docker/cli e2e tests
-func (s *DockerTrustSuite) TestBuildContextDirIsSymlink(c *check.C) {
+func (s *DockerSuite) TestBuildContextDirIsSymlink(c *check.C) {
```

Comment:
This looks like a CLI-only test. I'll port this to a cli unit test so you can remove it here.